### PR TITLE
Adds a 'different' Booze-O-Mat for Space Pirates

### PR DESCRIPTION
**Pull request by Tranquill6**
[Guidelines]: # Good enough for this server.

[Changelogs]: # Relates to Issue #9682 where Pirates weren't able to buy from their Booze-O-Mat due to the lack of a bank account, so a new machine was added that lets all alcohol be free.

[why]: They're Space Pirates, why would they charge on their own machine. At least now Pirates can use the machine until a better solution is suggested, but it takes too much effort to design coins for them to put in for alcohol for an antag that is barely used, only from events. Bank accounts do not make sense for Pirates too, so this works.

### CODE REVIEW

File: _maps/shuttles/pirate_default.dmm [hunk 1]
```diff
@@ -1,1488 +1,123 @@
-//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
```
Comment (Manly-Dwarf) [inline _maps/shuttles/pirate_default.dmm]: :spange:

Comment (yoyobatty) [reply _maps/shuttles/pirate_default.dmm]: Standard

**Comment by yoyobatty** [comment]
Did you forget to run map merge?

**Review by yoyobatty — APPROVED** [review]
Alright, looks good.

**Pull request merged**